templates/heka/telemetry/telemetry_payload_size.1.parquetmr.txt

```diff
@@ -4,6 +4,6 @@ message telemetry_payload_size {
     required binary build (UTF8);
     required binary channel (UTF8);
     required binary docType (UTF8);
-    required int64 size;
+    required double size;
   }
 }
```

Comment:
1. Proposed schema
```
message telemetry_payload_size {
  required int64 Timestamp;
  required int64 size; -- << add support for d2p output of size
  required group Fields {
    required binary build (UTF8);
    required binary channel (UTF8);
    required binary docType (UTF8);
  }
}
```
2. switch the budget dashboard to use this parquet output (the outputs can run in parallel during the migration)
3. remove moz_telemetry_payload_size.lua and its S3 protobuf stream output

Reply:
https://github.com/mozilla-services/lua_sandbox_extensions/commit/018f20dae29b776df22a663e4da138a8b2105213

Reply:
f+, this plan sounds good to me.

Reply:
Also sounds good to me. I don't think we need to worry about the switch over, since the dashboard isn't using this data yet anyways.

Reply:
For the record, the "dashboard" piece is [Bug 1370529](https://bugzilla.mozilla.org/show_bug.cgi?id=1370529), and we can follow up on that separately. Once we have this data in parquet form, we might be able to migrate that dashboard over to re:dash.